src/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities/Allegro.CosmosDb.BatchUtilities.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Allegro.CosmosDb.BatchUtilities</PackageId>
     <PackageDescription>Utilities for performing batch operations in Azure CosmosDb, such as rate limiting and autoscaling.</PackageDescription>
-    <Version>1.0.3</Version>
+    <Version>2.0.16</Version>
```

Comment:
why so high?

Reply:
for my testing, before completing, i will set 2.0.0